src/test/kotlin/io/github/mojira/arisa/modules/CrashModuleTest.kt

```diff
@@ -173,6 +173,69 @@ const val JAVA_CRASH = """#
 # See problematic frame for where to report the bug.
 #"""
 
+const val OBFUSCATED_CRASH = """---- Minecraft Crash Report ----
```

Comment:
Why do we need this and can't just reuse `EXAMPLE_CRASH` or something similar?

Also this test file in general is massive and from what I can tell also partially tests mc-crash-lib. Instead of putting the entire crash report here, we should probably just mock the crash result or something.

It's ok for this PR but test organization is something that I'd like to look at in the future, atm it's very unweildy to navigate the test files and that's not good.

Reply:
Tried reusing the other ones but none of them have the neccesary info (actually obfuscated classes)

Ideally I would move all this constants to actual files, or mock as you are saying